src/main/resources/Diagram/DiagramSheet.xml

```diff
@@ -48,4 +48,122 @@
   #end
 #end
 {{/velocity}}</content>
+  <object>
+    <name>Diagram.DiagramSheet</name>
+    <number>1</number>
+    <className>XWiki.JavaScriptExtension</className>
+    <guid>ad4bd9bd-2932-40c3-a7b9-5caae1266b43</guid>
+    <class>
+      <name>XWiki.JavaScriptExtension</name>
+      <customClass/>
+      <customMapping/>
+      <defaultViewSheet/>
+      <defaultEditSheet/>
+      <defaultWeb/>
+      <nameField/>
+      <validationScript/>
+      <cache>
+        <cache>0</cache>
+        <defaultValue>long</defaultValue>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
```

Comment:
This is not good. See https://github.com/xwikisas/application-licensing/issues/67#issuecomment-523871959 .